client/a8c-for-agencies/sections/sites/sites-dashboard/index.tsx

```diff
@@ -38,10 +37,12 @@ import { updateSitesDashboardUrl } from './update-sites-dashboard-url';
 import './style.scss';
 
 export default function SitesDashboard() {
-	useQueryJetpackPartnerPortalPartner();
```

Comment:
As mentioned https://github.com/Automattic/wp-calypso/pull/88646, we can remove this as we technically switched to a different authentication mechanism. Sharing this for documentation purposes only.